lib/ws-channel.js

```diff
@@ -132,7 +172,7 @@ WebsocketChannel.prototype.request = function(message, callback) {
 
   var packet = {
     type: REQ,
-    seqno: seqno,
```

Comment:
It's largely academic in this case, but we need to keep the protocol stable because different ends of this connection can/will be upgraded on different schedules.